pyuvdata/miriad.py

```diff
@@ -53,7 +53,7 @@ def read_miriad(self, filepath, correct_lat_lon=True, run_check=True, run_check_
                               'telescope_name': 'telescop',
                               'instrument': 'telescop',
                               'Nants_telescope': 'nants',
-                              'antenna_positions': 'antpos',  # take deltas
+            #                  'antenna_positions': 'antpos',  # take deltas
```

Comment:
I suggest deleting this line. You don't need to leave commented code around to know the history because with git you can always figure it out. And commented code clutters things up and makes it hard to read.